### DISCUSSION TITLE
opentelemetry-okhttp-3.0 has only alpha tags

### DISCUSSION THREAD

**Discussion by tadlakha9** [open-telemetry/opentelemetry-java-instrumentation#12367] [Q&A]
We have identified that 

```
// https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-okhttp-3.0
runtimeOnly("io.opentelemetry.instrumentation:opentelemetry-okhttp-3.0:2.5.0-alpha")
```

has only alpha tag, can't we have any stable version of this library.

**Comment by steverao** [comment]
I saw you file an issue https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12389, we can discuss in it later and close the discussion now.
